### DISCUSSION TITLE
Any way to prevent links from overlapping?

### DISCUSSION THREAD

**Discussion by uberhus** [kurkle/chartjs-chart-sankey#108] [Q&A] [answered]
First of all I want to thank @kurkle for creating this awesome module! I've been testing this and one thing I have not been able to resolve is the overlapping links, like here where the "Unsold 2", "Unsold 3" and "Unsold 4" overlap with other links even though they wouldn't have to: 

![image](https://github.com/kurkle/chartjs-chart-sankey/assets/34740599/04a653de-6be2-4747-9398-503d1851ed51)

Here's a codepen: https://codepen.io/Yli-T/pen/OJGXwKg

I tried to resolve this by setting priorities to all nodes, but it just got worse (you can see the result if you remove the comment before priority inside the dataset).

I love this module, but I think it would make the chart more readable if the links didn't overlap in places where they are not forced to do so. So what I am wondering is that is there any way to resolve this by changing some chart options, or is this a bigger development task? I can see that there hasn't been much development recently, but I hope someone still maintains this because, as I said, this module is awesome!

**Comment by kurkle** [comment, marked as answer]
Sorry for late answer, been doing other stuff lately.

But anyhow, do you think this kind of layout would work:
![image](https://github.com/user-attachments/assets/edf6f6dc-ef10-4112-a8ea-2b2e51c1c031)

**Reply by uberhus** [reply]
Hi @kurkle sorry for late answer as well. Yes that works perfectly, in fact I realised at some point after trying various options that I can do exactly what you are showing there if I use the column overrides. So the module works perfectly, thanks once again!
